source/software/singularity.rst

```diff
@@ -173,6 +173,38 @@ An important advantage of definition files is that they can easily
 be shared, and improve reproducibility.
 
 
+Conda environment in a definition file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Before showing how, I would propose to tell the users why then shall care about this block.
What is the motivation for the users to install conda in a container instead of e.g. in their VSC_DATA?
- One possible scenario is for groups who work collectively, and they use an identical conda environment (built from a reference YAML recipe). So, this approach lowers the total storage lost for the whole group only due to duplicate conda installations.
- Similarly, those users who share a single conda environment on staging are sometimes hit by slow response of conda (due to Lustre problems with too many small files), which might seldom exhibit itself

Reply:
I think that all of the above suggested edits fall under the already existing subsection
https://docs.vscentrum.be/software/singularity.html#when-should-i-use-containers
A concise explanation about why and where one would like to use a container. That is true especially for the the first bullet point. Regarding the second bullet point, I think that:
1) sharing environment falls under the "when-should-i-use-containers" section. One could propose to expand that one instead rather than adding new information on the why-and-when container use cases.
2) The slow Lustre is our problem and we need to solve that and not advertise container use as a workaround.

Reply:
This is a matter of choice, and I leave it to you to decide.

Reply:
Added some changes. Requesting re-review